Description: Resolve autohide with multiple docks
 When intellihide/autohide is used with multiple panels, some panels
 never appear when the mouse cursor is moved to the panel location.
 This patch has been created from the upstream commit rebased
 with the 18.04 code baseline
Origin: Commit e3f941105e7684e599415a22752b6d00ec524450
Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2019-01-12

Index: budgie-desktop-10.4+git20171031.10.g9f71bb8/src/panel/manager.vala
===================================================================
--- budgie-desktop-10.4+git20171031.10.g9f71bb8.orig/src/panel/manager.vala
+++ budgie-desktop-10.4+git20171031.10.g9f71bb8/src/panel/manager.vala
@@ -314,7 +314,7 @@ public class PanelManager : DesktopManag
         Gdk.Rectangle win = Gdk.Rectangle();
         Gdk.Rectangle pan = Gdk.Rectangle();

-        if (window == null) {
+        if (window == null && this.window_list.length() == 0) {
             return false;
         }

